### ISSUE TITLE
Hidden selects show as 0 width

### ISSUE THREAD

**Issue by johnjamesjacoby** [stuttter/wp-chosen#3]
If a select is targeted with Chosen, but that select is hidden, Chosen is invoked with a `0` width.

See: https://github.com/harvesthq/chosen/issues/1267

This is particularly troublesome with metaboxes that can be open and closed.

<img width="682" alt="screen shot 2016-02-23 at 3 34 47 pm" src="https://cloud.githubusercontent.com/assets/88951/13267467/fffe2366-da42-11e5-8c58-1b49fa8d0109.png">

**Comment by johnjamesjacoby** [comment]
Mocking what I wish we could do here:

```

	/* Metabox Visibility */
	$( '.postbox .hndle, .postbox .handlediv' ).on( 'click.postboxes', function() {
		var inside  = $( this ).parent().children( '.inside' ),
			selects = inside.find( 'select' ),
			resized = function() {
				selects.show().chosen( chosen_options ).hide();
			};

		resized();
	} );
```

**Comment by johnjamesjacoby** [comment]
Still no clear or obvious solution to this.

Might require an upstream patch to WordPress to fire an event when a metabox is opened.

**Comment by johnjamesjacoby** [comment]
Fixed in aa54fdc5d74669e2a8b8f168e4d5dfeec5f78248
